android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -1169,10 +1169,6 @@ class GeckoEngineSession(
         override fun onShowDynamicToolbar(geckoSession: GeckoSession) {
             notifyObservers { onShowDynamicToolbar() }
         }
-
-        override fun onGetNimbusFeature(session: GeckoSession, featureId: String): JSONObject? {
```

Comment:
@jonalmeida I went ahead and removed `onGetNimbusFeature` to save a step for [bug 1846074](https://bugzilla.mozilla.org/show_bug.cgi?id=1846074) because the second [GV patch](https://phabricator.services.mozilla.com/D184841) that deprecates this will start causing AC to have build blocking deprecation warnings here. Please let me know if you don't want to remove it just yet.

Reply:
Excellent! That sounds fine to me. Could you update the changelog to reflect both of these changes? The addition and removal?

Reply:
Added, thanks!